src/cljs/main/broadfcui/common/components.cljs

```diff
@@ -607,7 +607,7 @@
 
 (defn no-billing-projects-message []
   [:div {:style {:textAlign "center"}}
-   (str "You must have a billing project associated with your account to create a new workspace. ")
+   "You must have a billing project associated with your account to create a new workspace." [:br]
```

Comment:
I prefer wrapping in a div to a br. It's easier to follow the presentation logic back that way.